db/migrate/20220326181609_posts.rb

```diff
@@ -0,0 +1,12 @@
+class Posts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :Posts do |t|
+      t.references :User, null: false, foreign_key: true, index: true
```

Comment:
- [ ] <b>REQUIRED:</b> Remember for User, the project requires to use `author_id`. You can check it out in the given [ERD Diagram.](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) so we do it like this
```rb
t.references :author, foreign_key: { to_table: 'users' }